### PR TITLE
Try this

### PR DESCRIPTION
**Pull request by FireMario211**
Changes to use `PRIMARY KEY("id")` in init.sql

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
`DEFAULT 0`? why is that?

**Comment by FireMario211** [comment]
> `DEFAULT 0`? why is that?

Forgot to remove that, since you're using `NOT NULL`.

**Comment by b1tt0** [comment]
pardon, but it doesn't fix any risk issues as seen [here](https://github.com/Em1tt/amethyst.host/pull/33/checks?check_run_id=2568890570)

**Comment by b1tt0** [comment]
reviewed the workflow logs - the issue seems to be with **naming**, but not `PRIMARY KEY`. could you please rename `id` to something more explicit in all tables & files?

**Review by ghost — CHANGES_REQUESTED** [review]
keep the consistent spaces - this is my last request

**Review by ghost — APPROVED** [review]
thanks!

**Comment by b1tt0** [comment]
@Em1tt, waiting for your approval

**Comment by Em1tt** [comment]
Great job team!

**Pull request merged**
